pkg/config/control_plane.go

```diff
@@ -183,6 +185,11 @@ type ControlPlaneCache struct {
 	TTL Duration `json:"ttl"`
 }
 
+type ControlPlaneInsightCollector struct {
+	Schedule string `json:"schedule"`
+	Disable  bool   `json:"disable"`
```

Comment:
With this `disable` field, the Insight collector will be executed by default, right?
We should change to not running it by default.

So instead of adding this `disable` field, as my previous comment, it could be done by adding an `enable-insight-collector` flag into the `ops` cmd.

https://github.com/pipe-cd/pipe/pull/1287/files#diff-74a98bb30498ead5e79526f5310672756633100c8f2556257601d778d51b4a6bR53

Reply:
OK, I understand what you want to say.
Add flag on ops cmd.